.changelog/16a15922898f472db59dd9deb6922968.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id": "16a15922-898f-472d-b59d-d9deb6922968",
+    "type": "bugfix",
+    "description": "Fix confusing doc header in `aws/signer/v4` package.",
+    "modules": [
+        "."
```

Comment:
Should this be scoped more narrower?

Reply:
It can only target modules, not packages - v4 is part of the core `aws-sdk-go-v2` module.